recipe/meta.yaml

```diff
@@ -44,21 +44,22 @@ source:
     # from Reference-LAPACK/lapack#445, see also xianyi/OpenBLAS#2864
     - patches/0008-fix-dlanv2.patch
     # from Reference-LAPACK/lapack#391
-    - patches/pr391.patch
+    - patches/0009-remove-extraneous-semicolon.patch
+    # avoid linker errors on windows
+    - patches/0010-don-t-declare-RowMajorStrg-as-extern-on-windows.patch       # [win]
+    # make xerbla-shenanigans work on windows
+    - patches/0011-use-__declspec-selectany-for-cblas_xerbla-s-RowMajor.patch  # [win]
+    # keep things called libblas.dll & blas.lib (as with m2w64-compilers)
+    - patches/0012-keep-same-library-naming-as-before-switching-to-msvc.patch
```

Comment:
You can just set `-DCMAKE_SHARED_LIBRARY_PREFIX=lib` instead of patching.

Reply:
Nope, that did not work:
```
-- Installing: D:/bld/blas-split_1731017926582/_h_env/Library/lib/cblas.lib
-- Installing: D:/bld/blas-split_1731017926582/_h_env/Library/bin/cblas.dll
-- Installing: D:/bld/blas-split_1731017926582/_h_env/Library/lib/lapack.lib
-- Installing: D:/bld/blas-split_1731017926582/_h_env/Library/bin/lapack.dll
-- Installing: D:/bld/blas-split_1731017926582/_h_env/Library/lib/tmglib.lib
-- Installing: D:/bld/blas-split_1731017926582/_h_env/Library/bin/tmglib.dll
-- Installing: D:/bld/blas-split_1731017926582/_h_env/Library/lib/lapacke.lib
-- Installing: D:/bld/blas-split_1731017926582/_h_env/Library/bin/lapacke.dll
```

Reply:
Hmm, maybe it doesn't work from CLI. Try setting it in `CMakeLists.txt`?

Reply:
It's possible we'd need to backport https://github.com/Reference-LAPACK/lapack/commit/1c163c83fe41b4be415ef5aed24d8dafbfd8899d for this to work.